control_panel_api/tests/test_views.py

```diff
@@ -361,6 +369,87 @@ def test_customers(self, mock_get_group_members):
             set(response.data[0]),
         )
 
+    @patch('control_panel_api.auth0.auth0.api')
+    def test_post(self, mock_api):
+        api_get_user_response = {'user_id': 123}
+        mock_api.authorization.get.return_value = api_get_user_response
+
+        data = {'email': 'foo@example.com'}
+
+        response = self.client.post(
+            reverse('appcustomers-list', (self.fixture.id,)), data)
+
+        self.assertEqual(HTTP_201_CREATED, response.status_code)
+
+        mock_api.authorization.get_or_create.assert_called_with(
+            Group(name=self.fixture.name)
+        )
+
+        mock_api.authorization.get.assert_called_with(
+            Auth0User(email=data['email'])
+        )
+
+        mock_api.authorization.get_or_create.return_value.add_users.assert_called_with(
+            [{'user_id': api_get_user_response['user_id']}]
+        )
+
+    @patch('control_panel_api.auth0.auth0.api')
+    def test_post_create_user(self, mock_api):
+        api_create_user_response = {'user_id': 123}
+
+        mock_api.authorization.get.return_value = None
+        mock_api.management.create.return_value = api_create_user_response
+
+        data = {'email': 'foo@example.com'}
+
+        response = self.client.post(
+            reverse('appcustomers-list', (self.fixture.id,)), data)
```

Comment:
Just an observation, wouldn't be better to call this variable `app` instead of `fixture`?